### PR TITLE
Add SUSE PPC openssl 1.1.* support

### PR DESCRIPTION
**Pull request by JumpingYang001**
@microsoft/omi-devs , adds suse ppc with openssl 1.1.* support, the built binary will like `scx-1.6.3-1022.ssl_110.sles.12.ppc` and `scx-1.6.3-1022.ssl_110.sles.12.ppc.sh`, need review, thanks.

### CODE REVIEW

**Comment by sarojcare** [comment]
The changes done will not automatically build both the ssl version of ppc package. It need further investigation.

**Comment by JumpingYang001** [comment]
@sarojcare build 2 packages on same suse ppc box might need to follow the way with Universal build process like https://github.com/microsoft/ostc-openssl, we might need to do some investigation with building a box like Universal build server. Is SUSE PPC 12 fine to be our build server?
we will follow the way we done in Universal build, right?

File: build/Makefile.kits [hunk 1]
```diff
@@ -27,7 +27,16 @@ ifneq ($(COMBINED_PACKAGES),1)
     else
       ifeq ($(PF_DISTRO),SUSE)
         SCX_PREFIX=scx
-        OUTPUT_PACKAGE_PREFIX=$(SCX_PREFIX)-$(SCX_BUILDVERSION_MAJOR).$(SCX_BUILDVERSION_MINOR).$(SCX_BUILDVERSION_PATCH)-$(SCX_BUILDVERSION_BUILDNR).sles.$(PF_MAJOR).$(PF_ARCH)
+        SSLVERSION = `openssl version`
```
Comment (bbsaurav) [inline build/Makefile.kits]: Check if it possible to create a universal package for SLES 15 PPC

Comment (JumpingYang001) [reply build/Makefile.kits]: will try, thanks.

**Pull request closed without merging**